pkg/mumbleproto/Makefile

```diff
@@ -0,0 +1,6 @@
+Mumble.proto:
```

Comment:
Yes please. (OS X user)
On Mon, 28 Mar 2016 at 7:32 AM, Mikkel Krautz <notifications@github.com>
wrote:

> In pkg/mumbleproto/Makefile
> <https://github.com/mumble-voip/grumble/pull/6#discussion_r57534311>:
>
> > @@ -0,0 +1,6 @@
> > +Mumble.proto:
>
> I would prefer it if this used //go:generate instead.
>
> For example:
>
> generate.go:
>
> // -build
> //go:generate go run sync.go
>
> sync.go:
>
> // -build
>
> func main() {
> // fetch file
> // fix package name
> // write to disk
> }
>
> Then it would also work on Windows. (and OS X/BSD, where your sed change
> unfortunately wouldn't work:
> http://unix.stackexchange.com/questions/92895/how-to-achieve-portability-with-sed-i-in-place-editing
> )
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mumble-voip/grumble/pull/6/files/9d091cc32aeeb389b833587df0a525456c933a4c#r57534311>
>